scipy/signal/setup.py

```diff
@@ -1,5 +1,28 @@
 from scipy._build_utils import numpy_nodepr_api
 import os
+import sys
+
+def process_tempita(fromfile):
+    try:
+        try:
+            from Cython import Tempita as tempita
+        except ImportError:
+            import tempita
+    except ImportError:
+        raise Exception("Building requires tempita")
```

Comment:
Should we rely on `tools/npy_tempita` from NumPy instead?

Reply:
Comments crossed. We should vendor that, since `npy_tempita` does not get installed.

Reply:
ah yes. forgot about that.